systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -250,6 +251,8 @@ protected void changeClusterOperator(BundleVersionModificationData versionModifi
     }
 
     protected void copyModifyApply(File root, String namespace, final String strimziFeatureGatesValue) {
+        KubeClusterResource.getInstance().setNamespace(namespace);
```

Comment:
This is not related to the PR, but I think we should really remove options like this and specify namespace in every command that somehow interact with kube api. Partially it is done for fabric8 clients I think. IIUC here it could be passed to binding config directly or just to cmdKubeClient? WDYT?

Reply:
Yeah I know, I'm encountering stuff like this all the time. I have a list of things that should be done, but I don't want to do it in one PR. I wanted to add to all methods the specification of Namespace, but it's not related to this PR. I just needed to fix a problem when the Namespace is recreated and we are watching different one in tests. That is happening after my last PR with NamespaceManager. So it's a solution just for now, I will fix that in another PR if that makes sense.

Reply:
I think the NamespaceManager was good PR to implement it.

Reply:
Well yes, but it will need another full walkthrought the STs and a lot of lines of code in my opinion. I thought that it will be better to have just one PR for NamespaceManager and another one for adding that possibility